src/include/ipc/info.h

```diff
@@ -40,6 +40,16 @@ enum sof_ipc_ext_data {
 	SOF_IPC_EXT_WINDOW,
 };
 
+/* Compiler version */
+struct sof_ipc_cc_version {
+	char optimization; /* compiler -O flag value {'0','p','g','s'} */
+	char reserved[1];
+	uint16_t major;
+	uint16_t minor;
+	uint16_t micro;
+	char name[4]; /* null terminated compilator name */
```

Comment:
compiler name

Reply:
also thinking more, best to expand this to be 32bit for all version strings and a longer string. The reserved field could be bigger too.  e.g. Microsoft complier names are longer and tend to use large build numbers in versions so this would cover compilers that used similar conventions to Microsoft.

Reply:
done